src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/core/DocumentResourceIntTest.kt

```diff
@@ -166,17 +166,25 @@ class DocumentResourceIntTest : IntegrationTestBase() {
 
     @Test
     fun `unknown template name should return empty results`() {
-      webTestClient.get().uri("/documents/booking/$bookingId/template/NOT_FOUND")
+      webTestClient.get().uri("/documents/booking/$bookingId?templateName=NOT_FOUND")
         .headers(setAuthorisation(roles = listOf("ROLE_NOMIS_DOCUMENTS")))
         .exchange()
         .expectStatus().isOk
         .expectBody()
         .jsonPath("length()").isEqualTo(0)
     }
 
+    @Test
+    fun `will return error if only bookingId supplied`() {
+      webTestClient.get().uri("/documents/booking/$bookingId")
+        .headers(setAuthorisation(roles = listOf("ROLE_NOMIS_DOCUMENTS")))
+        .exchange()
+        .expectStatus().isBadRequest
+    }
+
     @Test
     fun `will return a document id filtered by bookingId and template name`() {
-      webTestClient.get().uri("/documents/booking/$bookingId/template/CSIPA1_FNP")
+      webTestClient.get().uri("/documents/booking/$bookingId?templateName=CSIPA1_FNP")
```

Comment:
would be nice to have a test where you're passing in multiple `templateName` parameters

Reply:
will add